### ISSUE TITLE
Installation of masonite-orm in Masonite project

### ISSUE THREAD

**Issue by girardinsamuel** [MasoniteFramework/orm#164]
After installing latest `masonite-orm` version in a new crafted Masonite 2.3.18 project and making require changes  (for migration) in `config/database.py` file and in `User.py` file, I ran into an error when running those commands in tinker:

```python
from app.User import User
User.all()
```
and I got 
```
booing parent scopes
boot methods []
running query: SELECT * FROM "users" ()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "/Users/samuel/.virtualenvs/masonite_test/lib/python3.8/site-packages/masonite/orm/models/Model.py", line 189, in all
    return cls.builder.set_action("select").all(query=query)
  File "/Users/samuel/.virtualenvs/masonite_test/lib/python3.8/site-packages/masonite/orm/builder/QueryBuilder.py", line 781, in all
    self.connection.query(self.to_qmark(), self._bindings) or []
  File "/Users/samuel/.virtualenvs/masonite_test/lib/python3.8/site-packages/masonite/orm/connections/PostgresConnection.py", line 108, in query
    raise e
  File "/Users/samuel/.virtualenvs/masonite_test/lib/python3.8/site-packages/masonite/orm/connections/PostgresConnection.py", line 98, in query
    cursor_factory=psycopg2.extras.RealDictCursor
AttributeError: module 'psycopg2' has no attribute 'extras'
```
I dit not ran migrations yet because of 1st problem and I have `psycopg2-binary==2.8.6`

Did I miss something when migrating to the new ORM ? 
Thanks 👍

**Comment by josephmancuso** [comment]
With all the refactoring and rewriting going on I haven't had a chance to make sure everything works outside of the unit tests. I'll take a look

**Comment by girardinsamuel** [comment]
Yes, sure I am also looking at it but it was just in case it was ringing a bell

**Comment by josephmancuso** [comment]
@girardinsamuel that actually looks like it must be a new psycopg2 release https://stackoverflow.com/questions/30940167/psycopg2-attributeerror-module-object-has-no-attribute-extras

**Comment by josephmancuso** [comment]
gonna mark this as a bug then if that's the case

**Comment by josephmancuso** [comment]
what version are you usin of psycopg2

**Comment by girardinsamuel** [comment]
psycopg2-binary 2.8.6

**Comment by josephmancuso** [comment]
that is a new release. I feel like I tested the Postgres database at least last month. I usually disable those tests locally but I wonder if 2.8.5 fixes it?

**Comment by josephmancuso** [comment]
> As of July 2020, the import psycopg2.extras works for me  

Maybe a new thing?

**Comment by girardinsamuel** [comment]
Yes I looked into it but, still same error with 2.8.5. 
But with this in mind I should be able to solve it, maybe that's **the** issue to get familiar with masonite-orm 👍

**Comment by girardinsamuel** [comment]
Here are the details of the release
https://www.psycopg.org/articles/2020/09/06/psycopg-286-released/

Nothing at first view about `extras`

**Comment by girardinsamuel** [comment]
Found it 
It's the way it's packaged

In a shell with 2.8.5 :

Not working (current)
```python
import psycopg2
psycopg2.extras.RealDictCursor
```

Working
```python
from psycopg2.extras import RealDictCursor
```

**Comment by girardinsamuel** [comment]
I'll make a PR

**Comment by josephmancuso** [comment]
should we open an issue with psycopg2? I feel like this shouldn't break between 2 minor versions ...

**Comment by girardinsamuel** [comment]
Actually I got this issue with 2.8.6, 2.8.5, 2.8.4 ...
You said it was working at one point ?

**Comment by josephmancuso** [comment]
I thought so lol. I guess not

**Comment by girardinsamuel** [comment]
Here is a PR which fixes it #165 👍 .

I edited the issue to put the other problem into its own issue.
